desktop/libs/azure/src/azure/abfs/abfs.py

```diff
@@ -572,6 +572,15 @@ def rename(self, old, new):
     Renames a file
     """
     headers = {'x-ms-rename-source': '/' + urllib_quote(Init_ABFS.strip_scheme(old))}
+
+    # Required to sign the header with SAS token for RAZ
+    if RAZ.IS_ENABLED.get():
+      raz_http_client = RazHttpClient(self._user, self._url, exc_class=WebHdfsException, logger=LOG)
```

Comment:
Reuse self._root._client? (not super clean but more chance to be consistent maybe)